pydantic_extra_types/country.py

```diff
@@ -4,28 +4,26 @@
 """
 from dataclasses import dataclass
 from functools import lru_cache
-from typing import Dict, List, Type, TypeVar
+from typing import Any, Dict, List, Type
 
-from pydantic import GetCoreSchemaHandler
+from pydantic import GetCoreSchemaHandler, GetJsonSchemaHandler
 from pydantic_core import PydanticCustomError, core_schema
 
 try:
-    import pycountry
-except ModuleNotFoundError:
+    import pycountry  # type: ignore[import]
+except ModuleNotFoundError:  # pragma: no cover
     raise RuntimeError(
         'The `country` module requires "pycountry" to be installed. You can install it with "pip install pycountry".'
     )
 
-T = TypeVar('T')
-
 
 @dataclass
 class CountryInfo:
     alpha2: str
     alpha3: str
     numeric_code: str
     short_name: str
-    # NOTICE: Not all countries have an official name
+    # NOTE: Not all countries have an official name
```

Comment:
`NOTE` highlights on VSCode. I'm not sure what PyCharm behavior is.

Reply:
🚀 integrations